django_tasks/backends/database/models.py

```diff
@@ -170,6 +170,13 @@ def task_result(self) -> "TaskResult[T]":
 
         return task_result
 
+    @property
+    def task_name(self) -> str:
+        try:
+            return self.task_path.rsplit(".", 1)[1]
```

Comment:
Thought: It'd be nice to keep using `task.name` by default, but falling back to these 2 if it's not available.